mdm/mdm/command.go

```diff
@@ -56,6 +56,7 @@ type Command struct {
 	SetAutoAdminPassword            *SetAutoAdminPassword
 	ScheduleOSUpdate                *ScheduleOSUpdate
 	ScheduleOSUpdateScan            *ScheduleOSUpdateScan
+	OSUpdateStatus                  *OSUpdateStatus
```

Comment:
This is not necessary to add a struct since it's `RequestType` with no additional keys.